git/hooks/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+files=`git diff --cached --name-status`
+re="gen-PKGBUILD.py"
+if [[ $files =~ $re ]]
+then
+  echo "Re-generating PKGBUILD..."
+  python gen-PKGBUILD.py > PKGBUILD
```

Comment:
What do you think about running `updpkgsums` after this and the removing the sums from the template?